src/commands/veloce/source/pull.ts

```diff
@@ -76,15 +76,15 @@ export default class Pull extends SfdxCommand {
     const {pmlsToDump, uisToDump} = Pull.spitMembers(members)
     const pmsToDump = new Set<string>()
 
-    const {pmlRecords, pmlPmsToDump} = await pullPml(sourcepath, conn, members === '', pmlsToDump)
+    const {pmlRecords, pmlPmsToDump} = await pullPml(this)(sourcepath, conn, members === '', pmlsToDump)
```

Comment:
if you are passing this in sych syntax, please tak sourcepath and conn from this and avoid using args for that.
if this is really improving things, not sure about this.
currently this is reduntant.